backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/WebService.kt

```diff
@@ -19,9 +20,10 @@ class WebService {
         val host = config.server.host
         val port = Integer.parseInt(config.server.port)
         val dataDirectory = config.server.dataDirectory
-
         val applicationData = File(dataDirectory, "applications")
 
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"))
+
```

Comment:
Can you check whether this is still necessary? Maybe we should remove it, if it's not necessary. (Just because we probably shouldn't fiddle with globals)

Reply:
this is necessary